app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryTitle.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const ExpandedStoryTitle = (props) => {
+
+  const { story, onEdit } = props;
+
+  return (
+    <div className="Story__section">
+      <div className="Story__section-title">
+        {I18n.translate('activerecord.attributes.story.title')}
+      </div>
+      <input
+        value={story._editing.title}
+        className="form-control input-sm"
+        onChange={(event) => onEdit({ title: event.target.value })}
```

Comment:
I think it would be better to pass just `onEdit(event.target.value)` here and change this (https://github.com/Codeminer42/cm42-central/pull/433/files#diff-dee40a0637f41a210d0b474cf5e31829R25) to be like this:

```js
onEdit={(newTitle) => editStory(story.id, { title: newTitle })}
```

You wouldn't expect that the "title" field pass an object with new attributes to its callback, right? You would expect only the new title.